frontend/src/index.html

```diff
@@ -6,91 +6,140 @@
+                    </div>
                   {%- endif %}
-                </div>
-              {%- endif %}
-              {% if audio.location -%}
-                <div>
-                  {{ audio.location }}
-                  {% if audio.locationUncertain -%}
-                    <span title="Location is uncertain">(?)</span>
+                  {% if audio.location -%}
+                    <div>
+                      {{ audio.location }}
+                      {% if audio.locationUncertain -%}
+                        <span title="Location is uncertain">(?)</span>
+                      {%- endif %}
+                    </div>
+                  {%- endif %}
+                  {% if audio.category -%}
+                    <div>{{ audio.category }}</div>
                   {%- endif %}
+                  {% for language in audio.languages -%}
+                    <div>{{ language }}</div>
+                  {%- endfor %}
                 </div>
-              {%- endif %}
-              {% if audio.category -%}
-                <div>{{ audio.category }}</div>
-              {%- endif %}
-              {% for language in audio.languages -%}
-                <div>{{ language }}</div>
-              {%- endfor %}
+              </div>
             </div>
-          </div>
+          </a>
+        {% endfor %}
+      </div>
+    </div>
+  </div>
+  <!-- Body left ends -->
+  <!-- Body Right -->
+  <div class="px-2 sm:w-full md:w-3/12 md:!pl-2 md:!pr-4">
+    {% for dailyvideo in dailyVideos offset: 0 limit: 2 %}
+      <div class="mb-8 ">
+        <div class=" pb-2 !text-sm font-semibold uppercase text-black md:text-base">Today's harikatha ({{ dailyvideo.language | language_name }})</div>
+        <div
+          x-data="{showthumb: true}"
+          class="aspect-h-9 aspect-w-16"
+        >
+          <template x-if="showthumb">
+            <img
+              @click="showthumb = false"
+              class="rounded-md"
+              src="https://img.youtube.com/vi/{{ dailyvideo.videoID }}/hqdefault.jpg"
+            >
+          </template>
+          <template x-if="!showthumb">
+            <iframe
+              class="h-auto w-full rounded-md"
+              src="https://www.youtube.com/embed/{{ dailyvideo.videoID }}?autoplay=1&enablejsapi=1"
+              allow="autoplay"
+            ></iframe>
+          </template>
         </div>
-      </a>
+      </div>
     {% endfor %}
+
+    <div class="mb-8">
+      <div class=" pb-2 !text-sm font-semibold  uppercase text-black md:text-base">short video of the day</div>
+      <div class="aspect-h-16 aspect-w-9">
+        <iframe
+          class="rounded-md"
+          src="https://www.youtube.com/embed/CwHT3zbTSI4"
+          title="Who experiences Separation"
+          frameborder="0"
+          allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share"
+          allowfullscreen
+        ></iframe>
+      </div>
+    </div>
   </div>
```

Comment:
We are not going to launch short videos now.

Reply:
short videos section removed